crates/pallet-balanced-currency-swap-bridges-initializer/src/lib.rs

```diff
@@ -113,16 +115,9 @@ pub mod pallet {
     pub type LastForceRebalanceAskCounter<T: Config> = StorageValue<_, u16, ValueQuery>;
 
     #[pallet::genesis_config]
+    #[derive(frame_support::DefaultNoBound)]
```

Comment:
Note that https://github.com/rust-lang/rust/issues/132162 is about to be stablilized